sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosEndToEndOperationLatencyPolicyConfigBuilder.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.models.CosmosEndToEndOperationLatencyPolicyConfig;
+
+import java.time.Duration;
+
+/**
+ * Builder for CosmosEndToEndOperationLatencyPolicyConfig
+ */
+public class CosmosEndToEndOperationLatencyPolicyConfigBuilder {
```

Comment:
```suggestion
public class CosmosEndToEndOperationConfigBuilder {
```

Reply:
```suggestion
public class CosmosEndToEndRequestConfigBuilder {
```

Reply:
I think the proposed one is super confusing - what is an endToEndOperation? either have EndToEndOperationTimeout or Latency - I am fine with removign the policy suffix

Reply:
Based on the discussions CosmosE2EOperationRetryPolicyConfig is the name